helm/charts/hydra/values.yaml

```diff
@@ -163,6 +163,14 @@ deployment:
 # Configure node affinity
 affinity: {}
 
+# Configure HPA
+autoscaling:
+  enabled: true
```

Comment:
Can we move this to the "deployment" top level key? Also let's disable this per default!

Reply:
sure